sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -971,7 +984,7 @@ func withProxyEnv() oci.SpecOpts {
 }
 
 // fetchImage returns a `containerd.Image` given an image source.
-func fetchImage(ctx context.Context, source string, client *containerd.Client, registryConfigPath string, useCachedImage bool) (containerd.Image, error) {
+func fetchImage(ctx context.Context, source string, client *containerd.Client, registryConfigPath string, useCachedImage bool, labels string) (containerd.Image, error) {
```

Comment:
Currently with this function as it is, the image will be labeled only when it doesn't exist, but if it does, the image won't be labeled. I think it doesn't hurt to always apply the labels to the image, regardless if they are already present (provided that containerd doesn't fail when the labels already exist for the image).

Reply:
I think we are fine. The current strategy can make sure we label all the images which needs to be labeled. I think it would be the same as label all images even they are already labeled.